bool    start
---
bool    end
---
sensor_msgs/BatteryState    battery_status
